### ISSUE TITLE
settings.js theme doesnt load on windows

### ISSUE THREAD

**Issue by Steve-Mcl** [node-red/node-red#3097]
### Current Behavior

Theme does not work (default node-red white theme still present). 

As a quick test, I disabled the path check [here](https://github.com/node-red/node-red/blob/f1e7ec0c6bc33d4d0f6286012ef4d5aa817ac0b5/packages/node_modules/%40node-red/editor-api/lib/editor/theme.js#L77) ...
```
            if (baseDirectory) {
                fullPath = path.resolve(baseDirectory,array[i]);
                //if (fullPath.indexOf(baseDirectory) !== 0) {
                //    continue;
                //}
            }
```

... and theme loads & works.

---

#### Additional details...
I added some `console.log`ging to `serveFilesFromTheme()` in `theme.js` like this...

```js
        for (var i=0;i<array.length;i++) {
            let fullPath = array[i];
            if (baseDirectory) {
                fullPath = path.resolve(baseDirectory,array[i]);
                console.log("baseDirectory", baseDirectory); <<< added logging
                console.log("fullPath", fullPath);<<< added logging 
                if (fullPath.indexOf(baseDirectory) !== 0) {
                    console.log("👎 NOT Gonna serve this file because  `fullPath.indexOf(baseDirectory) !== 0` "); <<< added logging
                    continue;
                }
            }
            var url = serveFile(themeApp,directory,fullPath);
            if (url) {
                result.push(url);
            }
        }
```

This is what I see in console...
```bash
baseDirectory \Users\Stephen\.node-red\node_modules\@node-red-contrib-themes\midnight-red
fullPath C:\Users\Stephen\.node-red\node_modules\@node-red-contrib-themes\midnight-red\theme.min.css
👎 NOT Gonna serve this file because  `fullPath.indexOf(baseDirectory) !== 0`
```

Basically, on windows, the theme file are not served because path.resolve returns the drive specifier where baseDirectory  does not.

### Expected Behavior

Theme to be applied

### Steps To Reproduce

On windows machine...
1.  Set  theme to "midnight-red" in `settings.js`
    ```js 
    editorTheme: {
        projects: {
            enabled: false
        },
        theme: "midnight-red",
        codeEditor: {
        }
    ```
2. restart node-red
3. refresh browser

### Example flow

N/A

### Environment

- Node-RED version: 2.0.5
- Node.js version: 14
- npm version: 6
- Platform/OS: Windows 10
- Browser: Chrome

**Comment by Steve-Mcl** [comment]
Hi devs,
I had an awful feeling this issue was with my system/installation but it turns out [another user on the forum](https://discourse.nodered.org/t/node-red-2-01-editor-theme/48774/38) also has same issue confirming what i found.

Ta.
